packages/main/test/pages/TabContainer.html

```diff
@@ -796,6 +796,14 @@ <h2>Text only Start And End Overflow Custom Overflow Buttons</h2>
 			<ui5-tabcontainer id="tabContainerEmpty">
 			</ui5-tabcontainer>
 		</section>
+		<section>
+			<h3>Add new selected tab programmatically</h3>
+			<ui5-tabcontainer id="tabContainerAddTabsProgrammatically">
+				<ui5-tab text="Tab 0"></ui5-tab>
```

Comment:
Make at one of those tabs selected, so that the code which resets the selection on line 883 will work - to replicate the real case. In the test, test the state of the tabs before the "Act".